packages/nimble-components/src/table/template.ts

```diff
@@ -47,14 +51,24 @@ export const template = html<Table>`
                             </${checkboxTag}>
                         </span>
                     `)}
-
+                    <span class="collapse-button-container" role="gridcell">
```

Comment:
nit: I'd vote to make class and variable names consistent between "collapse-all-button" and "collapse-button". No strong preference from me but it looks like you use "collapse all" in more places right now.